### ISSUE TITLE
Replace ComplexTypeSystem code borrowed from OPCFoundation with reference to their nuget package.

### ISSUE THREAD

**Issue by htbmw** [OpenManufacturingPlatform/iotcon-opc-ua-connector-dotnet#39]
Code was borrowed from OPCFoundation github a few years ago in order to apply custom fixes.
Need to check if issues still persist, and fix and create a PR for it on the OPCFoundation repo, so that we can reference the official nuget package instead of duplicating the code.
 - May need to create a fix and PR on OPCFoundation Github.

**Comment by htbmw** [comment]
I have replaced the borrowed OPCFoundation code with the Opc.Ua.Client.ComplexTypes nuget package and can confirm that the issue with recursive structs still persist.

I reported an issue on the OPCFoundation github repo:

https://github.com/OPCFoundation/UA-.NETStandard/issues/1655

As well as submitted a pull request for changes made to accommodate recursive structs:

https://github.com/OPCFoundation/UA-.NETStandard/pull/1656

This will be a work in progress, until OPCFoundation has accepted and released the changes.
Once released the nuget package will be updated and the connector be tested to ensure everything still works regarding complex types.
